amplipi/mpris.py

```diff
@@ -143,22 +144,22 @@ def close(self):
 
     self.metadata_process.terminate()
     if self.metadata_process.wait(1) != 0:
-      print('Failed to stop MPRIS metadata process, killing', flush=True)
+      logging.info('Failed to stop MPRIS metadata process, killing', flush=True)
```

Comment:
I can't find a reference to a `flush=true` in anything logger related. Can you find the definition for me?

Reply:
There's no such thing, I just missed that when converting the print to a log